engine/src/test/java/io/zeebe/engine/processing/CommandProcessorTestCase.java

```diff
@@ -47,8 +53,21 @@
   @Mock(name = "TypedStreamWriter")
   protected TypedStreamWriter streamWriter;
 
+  @Mock(name = "StateWriter")
```

Comment:
Nice, I didn't know one can name the mocks. That's a neat little feature

Reply:
Yeah same, I saw it being used in this class, so I continued. But I agree 👍